python/ql/src/Statements/IterableStringOrSequence.ql

```diff
@@ -13,15 +13,20 @@
 
 import python
 
+predicate has_string_type(Value v) {
+    v.getClass() = ClassValue::bytes() and major_version() = 2
+    or
+    v.getClass() = ClassValue::unicode()
+}
```

Comment:
Wait, so `v.getClass() = ClassValue::str()` doesn't work? The duplication of this logic bugs me a bit. Is this because `ClassValue::str` is too strict (in that it explicitly doesn't allow `unicode` for Python 2, unlike the above)?

Reply:
yep

Reply:
I guess you could instead do
```
v.getClass() = ClassValue::str()
or
v.getClass() = ClassValue::unicode()
```
I don't know if that's _less_ confusing, though. What do you think?

Reply:
How do you feel about this? It captures that we want what `str` is, but that for python 2 we will also allow unicode.

```
v.getClass() = ClassValue::str()
or
v.getClass() = ClassValue::unicode() and major_version() = 2
```

Reply:
I like it. :+1: :shipit:

Reply:
done